docs/sections/user_guide/cli/tools/fs.rst

```diff
@@ -166,15 +171,26 @@ Use the ``!glob`` tag to extract multiple members from one or more HPSS archive
 
 See :ref:`files_yaml_htar_glob_support` for more information on the use of the ``!glob`` tag in combination with ``htar://`` sources.
 
+``hardlink``
+------------
+
+The ``hardlink`` action stages items in a target directory by creating hardlinks to files or other links. It behaves similarly to the ``link`` action (see below), but note the following:
+
+* If a hardlink cannot be created but the ``--symlink-fallback`` option is specified, a symlink will be created instead, if possible. Hardlinks cannot target files on other filesystems, or directories.
```

Comment:
Is it worth mentioning here that hardlinks can't be created to files that you don't have write permission on? I don't know how common-knowledge hard links are in our user base.